### PR TITLE
Created the new C++ tutorial

### PR DESCRIPTION
**Pull request by Yeffian**

### CODE REVIEW

**Review by mellobacon — COMMENTED** [review]
There are some typos that needs to be addressed. Proofreading needed.

**Comment by Yeffian** [comment]
Fixed the errors, waiting for further reviews.

File: content/tutorials/cpp/00_setup_and_introduction.md [hunk 1]
```diff
@@ -9,84 +9,93 @@ series: "C++"
...
 ## What is C++?
-
-C++ is a general-purpose, OOP or __Object Oriented Programming language__. I will explain more about what OOP in a later post, as it is a slightly more advanced topic. It was developed by __Bjarne Stroustrup__, as a extension of the C programming language, or as some call it, "C with classes". C++ is most useful for systems programming, app development, and game development (Which is personally what I use it for).
-
-Ok, enough boring history talk, lets get started!
-
+ 
+C++ is a general-purpose, OOP or __Object Oriented Programming language__. I will explain more about OOP in a later post, as it is a slightly more advanced topic. It was developed by __Bjarne Stroustrup__, as an extension of the C programming language, or as some call it, "C with classes". C++ is most useful for systems programming, app development, and game development (Which is personally what I use it for).
+ 
+Ok, enough boring history talk, let’s get started!
```
Comment (mellobacon) [inline content/tutorials/cpp/00_setup_and_introduction.md:23]: > let’s get started

I don't think "let's" exists...

Comment (Yeffian) [reply content/tutorials/cpp/00_setup_and_introduction.md:23]: image

Comment (mellobacon) [reply content/tutorials/cpp/00_setup_and_introduction.md:23]: Oh. Carry on then

Comment (Yeffian) [reply content/tutorials/cpp/00_setup_and_introduction.md:23]: So uh, now what?

Comment (mellobacon) [reply content/tutorials/cpp/00_setup_and_introduction.md:23]: ANF hasn't seen the PR yet I don't think. Only he can merge so your job here is done unless he wants you to change something

Comment (Yeffian) [reply content/tutorials/cpp/00_setup_and_introduction.md:23]: Ok then.
Back to sleep I go, lets hope ANF doesn't take too long.

**Comment by anf** [comment]
@YeffyCodeGit, @mellobacon // I am a bit busy implementing a new theme in the `loveit` branch. As soon as that's merged with master, this pull request will be processed. You might want to look at its configuration and work something out with it. Transferring won't be hard since it's only one file. Until then, this pull shall stay on hold 🙂 

And you might want to also resolve conversations once they're done:
image

**Comment by anf** [comment]
@YeffyCodeGit // With pull #5 merged, you need to update it.

```md
---
title: "An introduction into programming and writing our first lines of code"
description: "Hey there! This is the very first article of this series in which we will be learning C# and working on projects."
slug: introduction
date: 2021-02-13T12:02:32-05:00
draft: false
aliases: 
- /blog/csharp/introduction
featuredImage: "thumb.png"
---

<!--more-->
```

You can copy something like this (including the more comment, that's also required)

**Comment by Yeffian** [comment]
What is this part?
`aliases: 
- /blog/csharp/introduction
featuredImage: "thumb.png"`

**Comment by anf** [comment]
I think you should have a look at [Hugo's Documentation](https://gohugo.io/documentation) and [Hugo LoveIt's documentation](https://hugoloveit.com/categories/documentation) as it'll be more efficient. You should have done this, instead.

However, let's not further waste time;
Aliases are alternate forms of the path e.g, if the page site is https://anf.github.io/how_to_really_learn_programming and we set an alias to it such as https://anf.github.io/foobar, the latter will __redirect__ that page to the former.
featuredImage is the thumbnail preview, as you did saw the site on the first C# series. You can leave that empty as it is managed by me.

**Comment by anf** [comment]
Yeff, you have yet to work on this. You are given a limited amount of time to work on this.

**Comment by mellobacon** [comment]
Uh didn't he resign?

**Comment by Yeffian** [comment]
Yeah, I did.
You should probably like, not let me see your stuff anymore.

**Pull request closed without merging**